gossip/tests/crds_gossip.rs

```diff
@@ -655,7 +655,9 @@ fn test_star_network_push_ring_200() {
     let thread_pool = build_gossip_thread_pool();
     network_simulator(&thread_pool, &mut network, 0.9);
 }
+
 #[test]
+#[ignore]
```

Comment:
should this come back before merging?

Reply:
Well, the test can be deleted because it is testing old pruning logic which is not true any more.
I am delaying that for now until this code is stable (i.e. settled down on optimal pruning once tested on testnet).

Reply:
let's add a comment, then

Reply:
added comment.